interview-questions.md

```diff
@@ -0,0 +1,67 @@
+# ASSESSMENT 6: Interview Practice Questions
+
+Answer the following questions.
+
+First, without external resources. Challenge yourself to answer from memory.
+
+Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
+
+1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
+
+Your answer: the foreign key would be called cohort_id and added to the student table. Then you rails db:migrate to add a column to the table.
+
+Researched answer: a foreign key is a column in a database that are linked to a column in a different table. The foreign key column will be cohort_id and the foreign key Student.
+
```

Comment:
Both are great responses! I encourage you to add a little more content. Show off your knowledge, tell us what steps we would take to add that new column.